src/js/types/networks/cardano.js

```diff
@@ -19,14 +19,23 @@ export type CardanoPublicKey = {
 
 // GetAddress
 
+export const CARDANO_ADDRESS_TYPE = Object.freeze({
+    Base: 0,
+    Pointer: 4,
+    Enterprise: 6,
+    Byron: 8,
+    Reward: 14,
+});
```

Comment:
I like this, though I would use uppercase and binary constants: ledger-js has
```
  BASE: 0b0000,
  POINTER: 0b0100,
  ENTERPRISE: 0b0110,
  BYRON: 0b1000,
  REWARD: 0b1110
```

Reply:
I'm not sure about both the uppercase and binary constants. Uppercase would then be used as `CARDANO_ADDRESS_TYPE.BASE` and `CARDANO_ADDRESS_TYPE.Base` seems a bit better to me.

Since this is client facing (at least I think the types can be used from the outside), I would leave it as is.